app/assets/javascripts/components/assignments/assignment.jsx

```diff
@@ -24,6 +24,8 @@ const Assignment = createReactClass({
     wikidataLabel: PropTypes.string
   },
   render() {
+    const isArticle = this.props.course.home_wiki.project !== 'wikidata';
```

Comment:
Here also, get the project from the `article` (after `const article = ...` is defined).